core/src/main/java/de/danoeh/antennapod/core/service/playback/LocalPSMP.java

```diff
@@ -317,12 +320,53 @@ private void resumeSync() {
                 float rightVolume = UserPreferences.getRightVolume();
                 setVolume(leftVolume, rightVolume);
 
+                int skipIntro = 0;
+                int skipEnd = 0;
+                if (media instanceof FeedMedia) {
+                    FeedMedia feedMedia = (FeedMedia) media;
+                    FeedPreferences preferences = feedMedia.getItem().getFeed().getPreferences();
+                    skipIntro = preferences.getFeedSkipIntro();
+                    skipEnd = preferences.getFeedSkipEnding();
+                }
+
                 if (playerStatus == PlayerStatus.PREPARED && media.getPosition() > 0) {
                     int newPosition = RewindAfterPauseUtils.calculatePositionWithRewind(
                         media.getPosition(),
                         media.getLastPlayedTime());
                     seekToSync(newPosition);
                 }
+                String skipMesg = "";
+                String skipIntroMesg = "";
+                if (playerStatus == PlayerStatus.PREPARED && media.getPosition() == 0) {
+                    if (skipIntro > 0) {
+                        int duration = getDuration();
+                        if (skipIntro * 1000 < duration) {
+                            seekToSync(skipIntro * 1000);
+                            skipIntroMesg = context.getString(R.string.pref_feed_skip_intro) + " " +
+                                    skipIntro + " " +
+                                    context.getString(R.string.time_seconds);
+                        }
+                    }
+                    if (skipEnd > 0) {
+                        int duration = getDuration();
+                        if (skipEnd * 1000 < duration) {
+                            skipMesg = context.getString(R.string.pref_feed_skip_ending) + " " +
+                                    skipEnd + " " +
+                                    context.getString(R.string.time_seconds);
+                        }
+                    }
+                    if (skipMesg != "") {
+                        skipMesg = skipIntroMesg + "\n" + skipMesg;
```

Comment:
Why do you display both numbers? I think we should not display more information than necessary.

Reply:
removed dup